### ISSUE TITLE
ORT is not correctly exporting Nvidia Apex FusedLayerNorm correctly

### ISSUE THREAD

**Issue by mstfbl** [microsoft/onnxruntime#10648]
**Describe the bug**
ORT is incorrectly exporting models that contain the Nvidia Apex operator `apex.normalization.fused_layer_norm` ([link](https://nvidia.github.io/apex/layernorm.html)).

**Urgency**
There is no hard deadline, as there is a workaround of uninstalling Nvidia Apex and/or removing the Apex FusedLayerNorm dependency. However, it would be nice if the bug was resolved sooner than later. 

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 18.04
- ONNX Runtime installed from (source or binary): Binary
- ONNX Runtime version:
- Python version: 3.8
- PyTorch version: 1.12.0.dev20220208+cu111
- Visual Studio version (if applicable): N/A
- GCC/Compiler version (if compiling from source): N/A
- CUDA/cuDNN version: CUDA 11.1/cuDNN 8.0.5
- GPU model and memory: NVIDIA Tesla V100

**To Reproduce**
- Describe steps/code to reproduce the behavior.

```python
try:
    from apex.normalization.fused_layer_norm import FusedLayerNorm
except ImportError:
    print("Better speed can be achieved with apex installed from https://www.github.com/nvidia/apex.")
    class FusedLayerNorm(nn.Module):
        def __init__(self, hidden_size, eps=1e-12):
            """Construct a layernorm module in the TF style (epsilon inside the square root).
            """
            super(FusedLayerNorm, self).__init__()
            self.weight = nn.Parameter(torch.ones(hidden_size))
            self.bias = nn.Parameter(torch.zeros(hidden_size))
            self.variance_epsilon = eps

        def forward(self, x):
            u = x.mean(-1, keepdim=True)
            s = (x - u).pow(2).mean(-1, keepdim=True)
            x = (x - u) / torch.sqrt(s + self.variance_epsilon)
            return self.weight * x + self.bias
```

- Attach the ONNX model to the issue (where applicable) to expedite investigation.
I've attached the ONNX models that were generated with and without Nvidia Apex. I've also attached a printout of the two models. 

Please note that in `graph-with-apex`, `input.4` reads:

> %input.4 : Half(*, *, *, strides=[393216, 768, 1], requires_grad=1, device=cuda:0) = com.microsoft::PythonOp[inplace=0, input_convention="dddcc", input_float_scalar_positions=[4], input_float_scalars=[1.0000000000000001e-05], input_int_tuple_begins=[0], input_int_tuple_positions=[3], input_int_tuples=[768], input_requires_grads=[1, 1, 1, 0, 0], input_tensor_ranks=[3, 1, 1], input_tensor_types=[10, 10, 10], name="FusedLayerNormAffineFunction", output_tensor_ranks=[3], output_tensor_requires_grads=[1], output_tensor_types=[10], training_mode=0](%196, %197, %198) # /opt/conda/envs/TorchNightly/lib/python3.8/site-packages/apex/normalization/fused_layer_norm.py:87:0

whereas in `graph-without-apex`, `input.4` reads:

> %input.4 : Float(*, *, *, strides=[393216, 768, 1], requires_grad=1, device=cuda:0) = onnx::Add(%190, %194) # /mnt/batch/tasks/shared/LS_root/jobs/datacachetest/azureml/2e3aab36-9db1-45e8-a526-8f436d9733c0/wd/azureml/2e3aab36-9db1-45e8-a526-8f436d9733c0/azure/ml/babel/models/alexander_src/tnlrv5/modeling_tnlrv5.py:219:0

[TNLR_models_with_and_without_apex.zip](https://github.com/microsoft/onnxruntime/files/8127941/TNLR_models_with_and_without_apex.zip)
[graph-with-apex.txt](https://github.com/microsoft/onnxruntime/files/8127960/graph-with-apex.txt)
[graph-without-apex.txt](https://github.com/microsoft/onnxruntime/files/8127961/graph-without-apex.txt)

**Expected behavior**
I expect both the Nvidia Apex FusedLayerNorm and the given FusedLayerNorm to behave identically.

**Comment by garymm** [comment]
In what sense is the model with apex incorrect?
graph-with-apex contains `com.microsoft::PythonOp`, which is how ORTModule handles custom autograd functions, which should work fine with ORTModule.

If we wanted to avoid `PythonOp`, we could in theory do that, but I don't think it should be necessary.

**Comment by ytaous** [comment]
@mstfbl - let's close this for now. I'll validate it again with newer version of ort. thx

**Comment by ytaous** [comment]
Related to ongoing investigation - https://github.com/microsoft/onnx-converters-private/issues/21
